manifests/profiles/default.yaml

```diff
@@ -29,4 +29,7 @@ spec:
       configValidation: true
     gateways:
       istio-ingressgateway: {}
-      istio-egressgateway: {}
\ No newline at end of file
+      istio-egressgateway: {}
+
+  meshConfig:
+    compliancePolicy: "fips-140-2"
```

Comment:
Another concern: should not make it default

Reply:
Not default. I was testing.